tiles/src/main/java/com/protomaps/basemap/Basemap.java

```diff
@@ -137,15 +139,21 @@ static void run(Arguments args) {
       .addGeoPackageSource("landcover", sourcesDir.resolve("daylight-landcover.gpkg"),
         "https://r2-public.protomaps.com/datasets/daylight-landcover.gpkg");
 
-    Downloader.create(planetiler.config()).add("ne", neUrl, nePath).run();
+    Downloader.create(planetiler.config()).add("ne", neUrl, nePath)
+      // .add("pgf-encoding", "https://api.github.com/repos/wipfli/pgf-encoding/zipball/e9c03fb", sourcesDir.resolve("main.zip"))
+      .add("qrank", "https://qrank.wmcloud.org/download/qrank.csv.gz", sourcesDir.resolve("qrank.csv.gz")).run();
     //      .add("qrank", "https://qrank.wmcloud.org/download/qrank.csv.gz", sourcesDir.resolve("qrank.csv.gz")).run();
 
     var tmpDir = nePath.resolveSibling(nePath.getFileName() + "-unzipped");
     var naturalEarthDb = NaturalEarthDb.fromSqlite(nePath, tmpDir);
     //    var qrankDb = QrankDb.fromCsv(sourcesDir.resolve("qrank.csv.gz"));
     var qrankDb = QrankDb.empty();
 
-    planetiler.setProfile(new Basemap(naturalEarthDb, qrankDb)).setOutput(Path.of(area + ".pmtiles"))
+    String pgfEncodingRepoHash = "e9c03fb";
+    FontRegistry fontRegistry = new FontRegistry(pgfEncodingRepoHash);
+    fontRegistry.loadFontBundle("NotoSansDevanagari-Regular", "1", "Devanagari");
```

Comment:
it feels like we should probably just make FontRegistry a Singleton referenced statically, since there is only ever one instance, right? 

Passing it as a dependency to each layer could increase the testability, but I don't see us ever needing to pass a mock FontRegistry into a layer for layer-specific behavior.

Reply:
The static Singleton pattern seems like a good solution in this case. Thank you for the tip.